src/vm/fieldmarshaler.h

```diff
@@ -1982,6 +1989,132 @@ class FieldMarshaler_DateTimeOffset : public FieldMarshaler
 
 #endif // FEATURE_COMINTEROP
 
+class FieldMarshaler_FixedBuffer : public FieldMarshaler
```

Comment:
is it possible to update FieldMarshaller_NestedValueClass marshaller instead of adding a new one? For me, These two marshaller are very similar.

Reply:
Possibly. I'll try to combine them.